### PR TITLE
clarify imagestreamtag descriptions

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@smarterclayton ptal
@luciddreamz fyi

File: pkg/oc/cli/describe/helpers.go [hunk 1]
```diff
@@ -340,8 +340,10 @@ func formatImageStreamTags(out *tabwriter.Writer, stream *imageapi.ImageStream)
 		}
 
 		switch {
-		case !hasSpecTag || tagRef.From == nil:
-			fmt.Fprintf(out, "  pushed image\n")
+		case !hasSpecTag:
```
Comment (smarterclayton) [inline pkg/oc/cli/describe/helpers.go]: This isn't empty spec tag.  This is "no spec tag"

Comment (smarterclayton) [reply pkg/oc/cli/describe/helpers.go]: Or "pushed image"

Comment (bparees) [reply pkg/oc/cli/describe/helpers.go]: updating here https://github.com/openshift/origin/pull/16198

**Pull request merged**